pod/main/views.py

```diff
@@ -103,14 +103,18 @@ def in_maintenance():
 def download_file(request):
     """Direct download of requested file."""
     if request.POST and request.POST.get("filename"):
-        filename = os.path.join(settings.MEDIA_ROOT, request.POST["filename"])
-        wrapper = FileWrapper(open(filename, "rb"))
-        response = HttpResponse(wrapper, content_type=mimetypes.guess_type(filename)[0])
-        response["Content-Length"] = os.path.getsize(filename)
-        response["Content-Disposition"] = 'attachment; filename="%s"' % os.path.basename(
-            filename
-        )
-        return response
+        post_filename = request.POST["filename"].strip('/')
```

Comment:
It seems that this fixes the issue of filenames starting with / but not filenames containing ..

Reply:
Correct ! we use now django form and filepathfield validation to prevent injection and get file form outside media path